src/MonitorProfiler/Stacks/StackSampler.cpp

```diff
@@ -81,7 +81,7 @@ HRESULT __stdcall StackSampler::DoStackSnapshotCallbackWrapper(FunctionID funcio
 
     std::shared_ptr<NameCache> nameCache = state->GetNameCache();
     TypeNameUtilities nameUtilities(state->GetProfilerInfo());
-    IfFailRet(nameUtilities.CacheNames(funcionId, frameInfo, *nameCache));
+    IfFailRet(nameUtilities.CacheNames(*nameCache, funcionId, frameInfo));
```

Comment:
nit: `funcionId` -> `functionId`

Reply:
I'll fix that separately since it needs to be fixed across multiple branches.